src/containers/TweetsWithReplies.tsx

```diff
@@ -1,19 +1,23 @@
+import { useDispatch, useSelector } from "react-redux";
+
 import SingleTweet from "../components/SingleTweet";
-import tweets from "../MOCKS/tweets";
+import { tweetsActions } from "../redux/reducers/tweets";
+import { AppDispatch, RootState } from "../redux/store";
+import { Tweet2 } from "../types";
 import InfiniteList from "./InfiniteList";
 
-const mockPagination = {
-  docs: tweets,
-  page: 1,
-  limit: 10,
-  hasNextPage: false,
-};
-
 export const TweetsWithReplies: React.FunctionComponent = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const posts = useSelector<RootState, RootState["tweets"]["feedTweets"]>(
+    (store) => store.tweets.feedTweets
+  );
```

Comment:
can we use specific tweets already as below
const posts = useSelector<RootState, RootState["tweets"]["feedReplies"]>(
    (store) => store.tweets.feedReplies
  );